### ISSUE TITLE
Issue with "cplusplus1"

### ISSUE THREAD

**Issue by elvin101** [DBraun/TD-JUCE#2]
Greetings from Poland!

First of all thanks for sharing this repo. I look for some reverb for TouchDesigner audio for my art school project.

Back to the point, after doing everything just like you wrote in _"README.md"_ (I have Windows 10 x64), I came across an issue when I have build in Release in Visual Studio. There is no sound in _"cplusplus1"_. Also I don't see a file called _"TD-JUCE-Reverb.dll"_ in directory _/Plugins_.

![image](https://user-images.githubusercontent.com/43068833/97032599-5f03e080-1562-11eb-84b0-d8757f7f25a0.png)

![image](https://user-images.githubusercontent.com/43068833/97032953-dfc2dc80-1562-11eb-8270-dc991e559071.png)

I would be grateful if you could help me with this. 

I am looking forward to hearing from you.

**Comment by DBraun** [comment]
Hello! Thanks for trying out the repo.

It's definitely a problem that there's no TD-JUCE-Reverb.dll in the plugins folder. Can you check that it isn't somewhere inside the `build` directory? Were there any errors when building in Release with Visual Studio? I just did the instructions again and got `Build: 4 succeeded, 0 failed, 0 up-to-date` on the first run.

**Comment by elvin101** [comment]
I can't find _Plugins_ directory anywhere on my computer, looked also in `C:\Users\myusername\Documents\Derivative\Plugins` but it isn't there.

Today I gave my issue one more shot, I have compiled individualy TD-JUCE-Reverb.dll and it has appeared!

![image](https://user-images.githubusercontent.com/43068833/97078620-a5525180-15ed-11eb-9e01-46794e8fff5e.png)

But unfortunately the _cplusplus1_ still doesn't work :(

![image](https://user-images.githubusercontent.com/43068833/97078679-05e18e80-15ee-11eb-8e95-b417d734a502.png)

Later I have deleted everything and done us you wrote. Still doesn't work. Would you mind looking on this?

`1>------ Build started: Project: ZERO_CHECK, Configuration: Release x64 ------
1>Checking Build System
2>------ Build started: Project: TD-JUCE, Configuration: Release x64 ------
2>Building Custom Rule C:/Users/Elvin/Documents/GitHub/TD-JUCE/CMakeLists.txt
2>include_juce_audio_basics.cpp
2>include_juce_audio_formats.cpp
2>include_juce_core.cpp
2>include_juce_data_structures.cpp
2>include_juce_dsp.cpp
2>include_juce_events.cpp
2>Generating Code...
2>   Creating library C:/Users/Elvin/Documents/GitHub/TD-JUCE/build/Release/TD-JUCE.lib and object C:/Users/Elvin/Documents/GitHub/TD-JUCE/build/Release/TD-JUCE.exp
2>TD-JUCE.vcxproj -> C:\Users\Elvin\Documents\GitHub\TD-JUCE\build\Release\TD-JUCE.dll
========== Build: 2 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========`

Also I am sending you _debug.txt_ if it matters. Sorry for trouble, I am from an art habitat and programming is for me something completly new. 
[debug.txt](https://github.com/DBraun/TD-JUCE/files/5432796/debug.txt)

**Comment by DBraun** [comment]
Yes you have to create `C:\Users\myusername\Documents\Derivative\Plugins` yourself, but that isn't a necessary step for this project.

The behavior I want to happen is for F5 to build all the projects. I've realized F5 is only building TD-JUCE and not projects such as TD-JUCE-Reverb. If instead you use the Build drop-down menu, you can build all solutions and then use F5 to debug.

![2020-10-24 13_18_17-TD-JUCE - Microsoft Visual Studio](https://user-images.githubusercontent.com/2096055/97092769-780ca000-15fb-11eb-8b4c-7735ba8da801.png)

I will leave this issue open until the F5 behavior is improved.

**Comment by elvin101** [comment]
Thank you for your reply.

_Build Solutions_ seems to have worked quiet good. Here is what have happened [Build.txt](https://github.com/DBraun/TD-JUCE/files/5433852/Build.txt). 4 succeeded, 1 skipped.

Later I debuged and it has launched TouchDesigner as before, but the situation in the project is still the same with _cplusplus1_. I don't know but maybe this files can help us: [Build Debug.txt](https://github.com/DBraun/TD-JUCE/files/5433858/Build.Debug.txt), [Debug Debug.txt](https://github.com/DBraun/TD-JUCE/files/5433859/Debug.Debug.txt)?

It seems we are close to the solution.

**Comment by DBraun** [comment]
Cool, I'm not too worried about the skipped one. Do you have `TD-JUCE-Reverb.dll` now in the repo's plugin folder? Once the TouchDesigner project opens do you toggle the "Unload plugin" parameter on the Reverb plugin CHOP? I keep it unloaded in case I want to make a change to the network before I load it.

**Comment by elvin101** [comment]
Oh yes, it works! Sorry I was everytime toggling it before, but yesterday it was middle of the night in Poland, I was tired and forgot to do that this time. Big thanks mate!
